packages/generator/tests/acceptance/test_plugin_generator.py

```diff
@@ -38,36 +39,37 @@ def test___command_line_args___create_measurement___render_without_error(
 
 
 def test___command_line_args___create_measurement_with_annotations___render_without_exception(
```

Comment:
change "render_without_exception" to "render_without_error"

Reply:
Updated to "render_without_error".